variables.tf

```diff
@@ -37,6 +37,12 @@ variable "config_output_path" {
   default     = "./"
 }
 
+variable "config_file_permissions" {
+  description = "File mode of the Kubectl config file. For security concerns this defaults to 0600."
```

Comment:
What security concerns? There's nothing sensitive in this file?

Reply:
That's option if:
* Someone wants to be paranoid
* Someone does not want to share details with other users
* Someone modifies his cluster and adds more sensitive data to it (i.e. someone puts his key in it - `client-key-data`)

Moreover, basic `~/.kube/config` file created by `kubectl` also does not contain any sensitive information until you add some. So to follow best practices let's assume generated file is safe enough for everyone just by design.

Reply:
@szczad do you need this or you think some will ?